FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -49,6 +49,11 @@ public DefaultClientIO(string secret, Uri endpoint, TimeSpan timeout)
             message.Content = dataString;
             message.Headers.Authorization = authHeader;
 
+            var lastSeen = LastSeen.Txn;
```

Comment:
what if you make `LastSeen` a class variable....

Reply:
That would share per client, but not across clients. The expected concurrency in these clients isn't clear to me from one to another, so I decided to make this global. Is per-client "better"?

Reply:
(I think I have that right... C# is also not at all my thing)

Reply:
well, given this is a library, make things global might make the users code unpredictable